examples/README.md

```diff
@@ -0,0 +1,31 @@
+# SD JWT Examples
+
+This directory contains examples of how to use the SD JWT(sd-jwt-js) library.
+
+## How to run the examples
+
+```bash
+pnpm install
+```
+
+## Run the example
+
+```bash
+pnpm run {example_file_name}
+
+#example
+pnpm run complex
+```
+
+### Example lists
+
+- complex : Example of issue, present and verify the complex data.
+- issue: Example of issue sd-jwt
+- present: Example of present sd-jwt
+- verify: Example of verify sd-jwt
+- validate: Example of validate sd-jwt
+- custom: Example of using custom hasher and salt generator for sd-jwt
```

Comment:
I would love to have an example using decoy as well.

Reply:
I added decoy example :)